packages/pages/src/upgrade/pages_updater.ts

```diff
@@ -0,0 +1,83 @@
+import { execSync } from "child_process";
+import fs from "fs";
+import path from "path";
+
+const pagesImportRegex = /"@yext\/pages\/components"/g;
+const replacementText = '"@yext/sites-components"';
+
+// Function to install npm packages
+const installPackages = (targetDirectory: string) => {
+  try {
+    execSync(
+      `npm install @yext/sites-components@latest @vitejs/plugin-react@4.0.4 vite@latest --prefix ${targetDirectory}`,
```

Comment:
We can't assume they are using npm. We should really add it to the package.json manually.